chinglee-iot/FreeRTOS-Cellular-Interface
# Update print log fmt

Description
-----------
* Add log format config

Test Steps
-----------
arm-none-eabi-gcc can define the following in cellular config
```
#define CELLULAR_LOG_FMT_SIZE "u"
#define CELLULAR_LOG_FMT_UINT32 "lu"
#define CELLULAR_LOG_FMT_INT32 "ld"
```

Checklist:
----------



Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
